src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -114,13 +129,13 @@ public CFAccessor cfAccessor(CFAccessorCache cfAccessorCache) {
 	}
 	
 	@Bean
-	public ReactiveTargetResolver reactiveTargetResolver() {
-		return new ReactiveTargetResolver();
+	public ReactiveTargetResolver reactiveTargetResolver(CFAccessor cfAccessor) {
+		return new ReactiveTargetResolver(cfAccessor);
 	}
 	
 	@Bean
-	public CachingTargetResolver cachingTargetResolver(ReactiveTargetResolver reactiveTargetResolver) {
-		return new CachingTargetResolver(reactiveTargetResolver);
+	public CachingTargetResolver cachingTargetResolver(ReactiveTargetResolver reactiveTargetResolver, @Value("${cf.cache.timeout.resolver:300}") int timeout) {
```

Comment:
This looks inconsistent to me: why not `PromregatorConfiguration promregatorConfiguration` here?

Reply:
This will be addressed by `CloudFoundryConfiguration` (since it starts with `cf.` instead of `promregator.`) 

You'll see that in a following PR.